plugins/huggingface/tests/test_huggingface_api.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+from unittest.mock import ANY, patch
+
+from modelgauge.auth.huggingface_inference_token import HuggingFaceInferenceToken
+from modelgauge.prompt import SUTOptions, TextPrompt
+from modelgauge.sut import SUTCompletion, SUTResponse
+from modelgauge.suts.huggingface_api import (
+    HuggingFaceChatParams,
+    HuggingFaceChatRequest,
+    HuggingFaceResponse,
+    HuggingFaceSUT,
+)
+
+
+@pytest.fixture
+def fake_sut():
+    return HuggingFaceSUT("fake_uid", "https://fake_url.com", HuggingFaceInferenceToken("fake_token"))
+
+
+@pytest.fixture
+def prompt():
+    return TextPrompt(
+        text="some text prompt",
+        options=SUTOptions(max_tokens=5, temperature=1.0, random="random"),
+    )
+
+
+@pytest.fixture
+def sut_request():
+    return HuggingFaceChatRequest(
+        inputs="some text prompt", parameters=HuggingFaceChatParams(max_new_tokens=5, temperature=1.0)
+    )
+
+
+def test_huggingface_api_translate_text_prompt_request(fake_sut, prompt, sut_request):
+    request = fake_sut.translate_text_prompt(prompt)
+
+    assert isinstance(request, HuggingFaceChatRequest)
+    assert request == sut_request
```

Comment:
I think this test depends on the two fixtures happening to use the same "some text prompt", which isn't obvious from the test. I think the long-term solution here is simplifying the domain model so it's easy to construct real examples. But in the short term I'd suggest making `prompt()` into a method that takes the text as a parameter, and asserting specific things on the `request` object that demonstrate your intent as to what bits are important to be there.

Reply:
That makes sense. I made some changes, let me know what you think